sunpy/net/base_client.py

```diff
@@ -153,6 +153,7 @@ class BaseClient(ABC):
     """
 
     _registry = dict()
+    isMetaClient = False
```

Comment:
Why is this cased the way it is?

Reply:
Does this mean a client supplies only metadata?

Reply:
Yes, it means it is a metadata client, and by default it is set to false. If true, fido will ignore fetch for it.